experimental/table/server/common/src/main/java/alluxio/table/common/udb/UdbTable.java

```diff
@@ -49,4 +49,10 @@
    * @return returns a proto representing the table
    */
   UdbTableInfo toProto() throws IOException;
+
+  /**
+   *
+   * @return true if the udbtable is connected
+   */
+  boolean isConnected();
```

Comment:
how is this used? When is something "not" connected?

Reply:
also old comment, will remove